votor/src/consensus_pool/stats.rs

```diff
@@ -24,11 +19,45 @@ pub(crate) struct ConsensusPoolStats {
 
     pub(crate) new_certs_generated: Vec<u32>,
     pub(crate) new_certs_ingested: Vec<u32>,
-    pub(crate) ingested_votes: Vec<u32>,
+    ingested_votes: VoteStats,
 
     pub(crate) last_request_time: Instant,
 }
 
+#[derive(Default)]
+struct VoteStats {
+    notarize: u64,
+    finalize: u64,
+    skip: u64,
+    notarize_fallback: u64,
+    skip_fallback: u64,
+}
+
+impl VoteStats {
+    fn increment(&mut self, vote: &Vote) {
+        match vote {
+            Vote::Notarize(_) => self.notarize = self.notarize.saturating_add(1),
+            Vote::NotarizeFallback(_) => {
+                self.notarize_fallback = self.notarize_fallback.saturating_add(1)
+            }
+            Vote::Skip(_) => self.skip = self.skip.saturating_add(1),
+            Vote::SkipFallback(_) => self.skip_fallback = self.skip_fallback.saturating_add(1),
+            Vote::Finalize(_) => self.finalize = self.finalize.saturating_add(1),
+        }
+    }
+
+    fn record(&self, header: &'static str) {
+        datapoint_info!(
+            header,
```

Comment:
There is only one caller at the moment - will there be more in the future ? If not, seems like we can just define the string inside `datapoint_info!()` instead of passing it as an arg to the `record()` function.

As a nit, I feel like I have seen `record()` used as the name for updating fields and `submit()` as the function that calls `datapoint_*`